plugins/copr.py

```diff
@@ -488,7 +489,7 @@ def _guess_chroot(self):
             else:
                 chroot = ("opensuse-leap-{0}-{1}".format(dist[1], distarch))
         else:
-            chroot = ("epel-%s-x86_64" % dist[1].split(".", 1)[0])
+            chroot = ("epel-{}-{}".format(dist[1].split(".", 1)[0], platform.machine()))
```

Comment:
> defaults to x86_64 when in this case it's actually an aarch64 system.

The use of `platform` is not ideal.  Can you please use a different mechanism already used for other distributions, and fallback to x86_64 if unable to detect?

Reply:
platform is part of the standard python library are we sure that's not ok?

Reply:
platform removed

Reply:
You already removed it, but platform is deprecated.
https://docs.python.org/3.5/library/platform.html#platform.linux_distribution
`Deprecated since version 3.5, will be removed in version 3.7.`

Reply:
That's good to know, I use this in at least one other python script I use regularly, but what I normally use is platform.machine() like in this line. That looks like it's not being deprecated and removed if I'm reading correctly? I've never used platform.linux_distribution, specifically.